login_page.py

```diff
@@ -1,32 +1,26 @@
 from common import *
-import dashboard_page as DashboardPage
 
 TITLE = 'Login'
-locator = {'loginButton': (By.ID, 'loginButton'),
-           'username': (By.NAME, '__ac_name'),
-           'password': (By.NAME, '__ac_password')}
+
+locator = {"loginField": "#username",
+            "passField": "#passwrd",
+            "submitBtn": "#loginButton"}
 
 @timed
 @assertPageAfter('Zenoss: Dashboard')
 def login(driver, url, username, password):
-    driver.get(url)
 
-    try:
-        element = 'loginButton'
-        timeout = 10
-        element = WebDriverWait(driver, timeout).until(
-            EC.presence_of_element_located(locator[element]))
-    except TimeoutException:
-        print "{} was not loaded in {} secs.".format(element, timeout)
-        return False
+    # assumes already on the login page
+    login_field = find(driver, locator["loginField"])
+    pass_field = find(driver, locator["passField"])
+    submit_btn = find(driver, locator["submitBtn"])
```

Comment:
these are using the implicit wait feature. if elements cant be found, a timeout will occur here. In the future we can get rid of implicit waits by updating the `find` function, so implicit or explicit waits are safely separated.